supporting_content/dimensions/servicecategory.md

```diff
@@ -0,0 +1,82 @@
+# Column: ServiceCategory
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                                     |
+| --------- | ------------------------ | ------------------------------------------ |
+| AWS       | CUR                      | None                                       |
+| GCP       | Big Query Billing Export | None                                       |
+| Microsoft | Cost details             | ServiceFamily is close, but based on usage |
+
+## Discussion / Scratch space
+
+Principles behind Service Category, granularity, when to add, when to split etc.
+
+- Every category must represent a single technology. Do not "and" multiple categories together unless they are synonymous.
+- Each category should be unique and should not be a logical child of another category (e.g., Containers are a subclassification of Compute).
+- Categories are used for cost classification purposes only. Do not create new categories for marketing differentiation (e.g., Containers).
+- Avoid unnecessary words. Prefer short, concise names.
+- Each service should have 1 and only 1 service category that best aligns to its architectural design.
```

Comment:
When referring to this column shouldn't it use the Proper Name? e.g.  Service Category instead of service category